npm/react/examples/react-scripts/src/RemotePizza.cy-spec.js

```diff
@@ -10,7 +11,7 @@ describe('RemotePizza', () => {
       'pizza',
     )
 
-    mount(<RemotePizza />)
+    mount(<RemotePizza />, { ReactDom })
```

Comment:
Why is the `ReactDom` instance different?

Reply:
I described this in slack channel. It is because mount function is in the upper directory and linked using file:../

Reply:
👍